src/sentry/tasks/store.py

```diff
@@ -474,6 +476,7 @@ def _do_save_event(
     event = None
     try:
         manager = EventManager(data)
+        # event.project.organization is populated after this statement.
```

Comment:
Can someone confirm at this statement is true because of: https://github.com/getsentry/sentry/blob/master/src/sentry/event_manager.py#L475-L476